tests/unit_tests/camera/test_single_photon_electron_spectrum.py

```diff
@@ -21,6 +22,7 @@ def spe_spectrum():
         "max_amplitude": 1.0,
         "afterpulse_spectrum": None,
         "input_spectrum": "input_spectrum",
+        "afterpulse_amplitude_range": [4.0, 42.0],
```

Comment:
Range is different (but consistent with previous PR)

Reply:
I wasn't worried about using different values, just making sure it is not a mistake (and connected to the point about what this range actually is).